conmon-rs/server/src/container_log.rs

```diff
@@ -61,14 +71,14 @@ impl ContainerLog {
         .collect::<Result<Vec<_>>>()?;
         Ok(())
     }
-
-    /// Reopen the container logs.
+      /// Reopen the container logs.
     pub async fn reopen(&mut self) -> Result<()> {
         join_all(
             self.drivers
                 .iter_mut()
                 .map(|x| match x {
                     LogDriver::ContainerRuntimeInterface(ref mut cri_logger) => cri_logger.reopen(),
+                    LogDriver::Json(ref mut json_logger) => json_logger.reopen(),
```

Comment:
hey @saschagrunert  I am having a mismatched type error showing here ``match` arms have incompatible types
  --> conmon-rs/server/src/container_log.rs:81:61
   |
79 |                   .map(|x| match x {
   |  __________________________-
80 | |                     LogDriver::ContainerRuntimeInterface(ref mut cri_logger) => cri_logger.reopen(),
   | |                                                                                 ------------------- this is found to be of type `impl futures::Future<Output = std::result::Result<(), anyhow::Error>>`
81 | |                     LogDriver::Json(ref mut json_logger) => json_logger.reopen(),
   | |                                                             ^^^^^^^^^^^^^^^^^^^^ expected future, found a different future
82 | |                 })
   | |_________________- `match` arms have incompatible types` 
   and `await` on both do not remove the the type error (as the compiler suggests) , so what needs to be change in order to implement the type or should there be change in resullt type of crilogger ? (idk)

Reply:
did https://github.com/containers/conmon-rs/pull/1744#pullrequestreview-1661525911 change the error output at all?